aggregator/src/aggregation/circuit.rs

```diff
@@ -63,6 +66,8 @@ impl AggregationCircuit {
         snarks_with_padding: &[Snark],
         rng: impl Rng + Send,
         batch_hash: BatchHash,
+        g2: &G2Affine,
```

Comment:
breaking changes in circuit API @silathdiir 

`g2` and `s_g2` are from parameters